collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,311 @@
+With this new feature, people are able to activate the necessary <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>/
+<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers without having access to the server. You'll need to
+export your game as a Progressive Web App though.
+
+![Ensure Cross-Origin Isolation Headers export option](/storage/blog/progress-report-web-export-in-4-3/ensure-cross-origin-isolation-headers.webp)
+
+This is because this export option setup the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
+build, to intercept the requests from the server and inject those missing
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
+
+## Boot splash screens
+Just before the beta feature freeze deadline, [@patwork](https://github.com/patwork), a new
+[#web](https://chat.godotengine.org/channel/web) contributor,
+[managed to sneak up something really nice](https://github.com/godotengine/godot/pull/91128):
+
+He cleaned up the game page code and he updated the progress bar to use the semantic (and very useful) HTML `<progress>`
+tag.
+
+But, mainly and from now on, he made boot splash screens set in the project settings to be displayed on the screen
+while the engine and the assets are getting loaded by the browser.
+
+![New Godot Web loading screen](/storage/blog/progress-report-web-export-in-4-3/loading-screen.gif)
+
+If no boot splash screen has been set, the classic Godot boot splash will be seen, as it is currently displayed on
+other platforms.
+
+# What now?
+The <abbr title="Game Developers Conference">GDC</abbr> made me realize that my todo list as the Web lead was bigger than expected.
+
+Don't worry, it's only the beginning for the Godot Engine to truly offer game developers a great and reliable way to
+create Web games.
+
+So work will be done to finalize and stabilize the samples playback for the Web. In the future, the node chain system
+should be powerful enough to support audio effects on buses, so we intend to work on this too.
+
+Based on the feedback we received at the <abbr title="Game Developers Conference">GDC</abbr> from partners and friends,
+we know that we need a way to reduce the size of our exports. Currently, the 4.3 release Web build .wasm is around 50MB.
+This is the engine only, without any assets. This may feel small nowadays, especially with gigabytes we're used to see
+on desktop and console games, but it is way too large for Web "instant games".
+
+We have a few ideas in mind to address this, and it could even help optimize builds for other platforms!
+
+But if you know C/C++, you're familiar with the Web APIs and you feel contributing,
+don't hesitate to join our
+[#web team](https://chat.godotengine.org/channel/web) on the
+[Godot Engine developers chat](https://chat.godotengine.org).
```

Comment:
```suggestion
# Next steps
Don't worry, these changes only mark the beginning for the Godot Engine to truly offer game developers a great and reliable way to create Web games.

So work will be done to finalize and stabilize the samples playback for the Web. In the future, the node chain system
should be powerful enough to support audio effects on buses, so we intend to work on this too.

Based on the feedback we received at the <abbr title="Game Developers Conference">GDC</abbr> from partners and friends,
we know that we need a way to reduce the size of our exports. Currently, the 4.3 release Web build .wasm is around 50MB.
This is the engine only, without any assets. This may feel small nowadays, especially with gigabytes we're used to see
on desktop and console games, but it is way too large for Web "instant games".

We have a few ideas in mind to address this, and it could even help optimize builds for other platforms!

If you know C/C++, you're familiar with the Web APIs and you feel contributing,
don't hesitate to join our
[#web team](https://chat.godotengine.org/channel/web) on the
[Godot Engine developers chat](https://chat.godotengine.org).
```